KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/LiveStreamSettingsModule.java

```diff
@@ -151,9 +153,13 @@ public void onAppStart(IApplicationInstance applicationInstance) {
 
 	public void onStreamCreate(IMediaStream stream) {
 
-		if(stream.getClientId() < 0){ //transcoded rendition
+		//WMSProperties properties = stream.getProperties();
+		//KalturaStreamType type = (KalturaStreamType) properties.getProperty(Constants.KALTURA_STREAM_TYPE);
+		//boolean isRTSP = (type != null && type == KalturaStreamType.UNKNOWN_STREAM_TYPE.RTSP);
+
+		/*if(stream.getClientId() < 0 && !isRTSP){ //transcoded rendition
 			return;
-		}
+		}*/
```

Comment:
why not : why not checking (type == KalturaStreamType. KALTURA_STREAM_TYPE.RTSP && stream.getRTPStream()== null  OR(type == KalturaStreamType. KALTURA_STREAM_TYPE.RTMP && stream.getClientId() < 0 )?